eng/tests/UpToDate.ps1

```diff
@@ -0,0 +1,72 @@
+# This script verifies that subsequent calls to `build.cmd` don't cause js/css to be unnecessarily rebuilt.
+
+[CmdletBinding(PositionalBinding=$false)]
+param (
+    [string][Alias('c')]$configuration = "Debug",
+    [parameter(ValueFromRemainingArguments=$true)][string[]]$properties
+)
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference = "Stop"
+
+try {
+    $RepoRoot = Join-Path $PSScriptRoot ".." | Join-Path -ChildPath ".." -Resolve
+    $BuildScript = Join-Path $RepoRoot "build.cmd"
+
+    # do first build
+    & $BuildScript -configuration $configuration @properties
+    if ($LASTEXITCODE -ne 0) {
+        Write-Host "Error running first build."
+        exit 1
+    }
+
+    # list of sentinel files to check
+    $FileList = @(
+        "$RepoRoot\MLS.Agent\wwwroot\api\trydotnet.min.js",
+        "$RepoRoot\MLS.Agent\wwwroot\client\bundle.js",
```

Comment:
client can have a number of bundle.js files not just one

Reply:
What's the likelihood of `1.bundle.js` (or any of the others) being re-built and `bundle.js` not?  We can certainly update this to check everything under `wwwroot/`, but I figured I'd initially just check a few files.  Runtime doesn't matter much if we check 3 or 300 files.

Reply:
Turns out there are only 73 files under `wwwroot/` so I just updated it to check them all.